.gitlab-ci.yml

```diff
@@ -193,6 +193,7 @@ test_rust:
     - cd subprojects/gdk_rust
     - DEBUG=1 ./launch_integration_tests.sh bitcoin
     - DEBUG=1 ./launch_integration_tests.sh liquid
+    - DEBUG=1 ./launch_integration_tests.sh subaccounts
```

Comment:
please add `label` test as well

Reply:
Added in https://github.com/Blockstream/gdk/pull/93/commits/8496a1de0c97c2cfbca06c40deaf14fd8c7297f0